strings.txt

```diff
@@ -352,3 +352,30 @@ PLUGIN_QOBUZ_PROGRESS_READ_PLAYLISTS
 	EN	Fetching Playlists...
 	FR	Récupération des listes de lecture...
 	NL	Afspeellijsten ophalen...
+	
+PLUGIN_QOBUZ_USE_CLASSICAL_ENHANCEMENTS
+	EN	Enhanced classical music display
+
+PLUGIN_QOBUZ_USE_CLASSICAL_ENHANCEMENTS_DESC
+	EN	Choose Yes here if you want the plugin to use Qobuz data to attempt to:<br />* Show a playlist and subheadings for each composer/work within a release.<br />* Add the composer and work names to the track title in the playlist and now playing displays (the playlist cannot show subheadings).
+	
+PLUGIN_QOBUZ_CLASSICAL_GENRES
+	EN	Additional classical genres
+	
+PLUGIN_QOBUZ_CLASSICAL_GENRES_DESC
+	EN	If you find a release for which the enhanced display does not show, add the Qobuz genre (shown in the release listing) to the "additional genres" list here (comma separated if more than one).
+	
+PLUGIN_QOBUZ_USE_CLASSICAL_ENHANCEMENTS_TEXT
+	EN	<br /><br />For questions/bug reporting <a href="https://forums.slimdevices.com/forum/user-forums/3rd-party-software/94305-qobuz-com-streaming-plugin" target="_blank">visit the forum</a>.<br /><br /><b>Note:</b> for best results, use the "Material Skin" plugin. Otherwise, if using the default interface, do not use "large artwork" (option at bottom left of screen), the display will not be rendered well if you do. 
```

Comment:
Instead of including HTML in the string table you could use line feeds and have them replaced in the HTML template using the [`html_break`](http://www.template-toolkit.org/docs/manual/Filters.html#section_html_break_html_para_break) filter. That would give you the flexibility (if ever needed) to use the same strings in Squeezeplay or some other non-HTML interface.

Reply:
Can you point me towards a usage of this?

Reply:
The blue `html_break` actually is the link to the documentation 😉. 

http://www.template-toolkit.org/docs/manual/Filters.html#section_html_break_html_para_break

Reply:
OK, two questions on this:
1. Is it possible to apply a filter to the "desc" text string within a [% WRAPPER...]?
2. How would you deal with adding <b> and <a href...>? Separate strings?

Reply:
Ok, you're right. My suggestion probably was a bit over-engineered. Never mind 😁.

`_DESC` elements would even be harder indeed.